homework6/task.py

```diff
@@ -0,0 +1,36 @@
+phone_book_list = [
+    ["Kate", "+096231453"],
+    ["Jack", "+097312414"],
+    ["Alice", "+063672317"]
+]
+phone_book = dict(phone_book_list)
+stats = len(phone_book)
```

Comment:
this variable should be updated every time when structure of the phonebook is changed. It is better to put it after 'stat' command or use `len(phone_book)` there